.circleci/config.yml

```diff
@@ -25,7 +25,7 @@ jobs:
           transport.host: localhost
           TZ: "Etc/UTC"
     environment:
-      PYTHON_VERSION: 3.7
+      PYTHON_VERSION: 3.8
```

Comment:
which python version is being used in the instances?

Reply:
The expertise instance is using 3.7 right now so we'd have to do what Carlos did when he updated the matcher's Python version